Assets/Code/Scripts/Utils/NotifyingVariable.cs

```diff
@@ -27,7 +27,7 @@ public NotifyingVariable(T initialValue)
             Value = initialValue;
         }
 
-        private void SetValue(T value)
+        public void SetValue(T value)
```

Comment:
This shouldn't need to be public. It's invoked as a result of assigning Value. IE You should just be able to do `Position.Value = positionValue` in Thumbwheel.cs, which will trigger the set => SetValue(value) function.

Reply:
pushed up a commit to address this